var/spack/repos/builtin/packages/neurodamus-core/package.py

```diff
@@ -14,7 +14,7 @@ class NeurodamusCore(Package):
     version('develop', git=git, branch='master')
     version('2.0.0', tag='2.0.0')
 
-    variant('python', default=True, description="Enable Python Neurodamus")
+    variant('python', default=False, description="Enable Python Neurodamus")
```

Comment:
we can also install this by default?

Reply:
As discussed, I still want to test neurodamus-py better.
Other issues include: 
 - Have python dependencies all available in spack streams
 - implement (at least some) of the missing features compared to hoc

Reply:
ok understood. fine.